html/global_attributes.json

```
@@ -743,6 +743,46 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "ignores_find_in_page": {
+          "__compat": {
+            "description": "Element is ignored for the purposes of find-in-page.",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLElement/inert",
```

I don't see any useful docs on find-in-page the HTMLElement page. 

```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/inertt",
```

---

Removal also works for me.

---

```suggestion
```